### PR TITLE
Create zh_cn.lang

### PR DESCRIPTION
**Pull request by Snownee**
Purpose
---
Added Chinese translations for Pepis

Reason
---

### CODE REVIEW

File: src/main/resources/assets/jp.crafterkina.pipes/lang/zh_cn.lang [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+advancement.jp.crafterkina.pipes.general.buildProcessorBase.title=无需红石
+advancement.jp.crafterkina.pipes.general.buildProcessorBase.desc=用石头和木棍制作一枚处理器板
+advancement.jp.crafterkina.pipes.general.buildAccelerationProcessor.title=加点糖！
+advancement.jp.crafterkina.pipes.general.buildAccelerationProcessor.desc=用处理器板和糖制作一枚加速器
+advancement.jp.crafterkina.pipes.general.buildAccelerationProcessorOver.title=太甜了？
+advancement.jp.crafterkina.pipes.general.buildAccelerationProcessorOver.desc=制作一枚x6.75以上的加速器
+advancement.jp.crafterkina.pipes.general.buildAccelerationProcessorUltimate.title=霓虹制糖厂
+advancement.jp.crafterkina.pipes.general.buildAccelerationProcessorUltimate.desc=制作一枚x100以上的加速器
+advancement.jp.crafterkina.pipes.general.buildExtractionProcessor.title=节省空间
+advancement.jp.crafterkina.pipes.general.buildExtractionProcessor.desc=用处理器板和木板制作一枚加速器
```
Comment (CrafterKina) [inline src/main/resources/assets/jp.crafterkina.pipes/lang/zh_cn.lang]: 提取器

File: src/main/resources/assets/jp.crafterkina.pipes/lang/zh_cn.lang [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+advancement.jp.crafterkina.pipes.general.buildOnewayProcessor.title=记录支持
+advancement.jp.crafterkina.pipes.general.buildOnewayProcessor.desc=用处理器板和铁锭制作一枚加速器
```
Comment (CrafterKina) [inline src/main/resources/assets/jp.crafterkina.pipes/lang/zh_cn.lang]: 单向

**Review by CrafterKina — CHANGES_REQUESTED** [review]
Thanks!
There seems to be some copy and paste mistakes.

**Comment by Snownee** [comment]
Corrected

**Review by CrafterKina — APPROVED** [review]

**Pull request merged**